plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -815,6 +815,13 @@ fu_cros_ec_usb_device_write_firmware (FuDevice *device,
 								     firmware,
 								     section,
 								     error)) {
+				if ((*error)->code == G_USB_DEVICE_ERROR_NOT_SUPPORTED) {
+					g_debug ("failed to transfer section, trying another write, ignoring error: %s",
+						 (*error)->message);
+					fu_device_add_flag (device, FWUPD_DEVICE_FLAG_ANOTHER_WRITE_REQUIRED);
```

Comment:
@hughsie I wonder if we actually want a different flag for this. My logic is this another write required is behind used differently than others do. This is another write required based on device come and go. Other times we use same device but make another pass.

I want to make sure we acknowledge the difference in the daemon and don't paper over this some day by changing behavior coming in and out of write with a replug.

Reply:
FWIW, the way that I'm using ANOTHER_WRITE_REQUIRED is always in conjunction with a device reboot or some kind of event that results in a reenumeration.

It's possible I'm the only one using it in this way, as last year I filed #2297, where it was broken and wouldn't actually survive the go-around in the detaching and reenumerating case.